core/src/test/scala/matryoshka/spec.scala

```diff
@@ -402,6 +402,34 @@ class FixplateSpecs extends Specification with ScalaCheck with ScalazMatchers {
       }
     }
 
+    def depth[T[_[_]], F[_]]: (Int, T[F]) => Int = (i, t) => i + 1
```

Comment:
Ah, of course. And the ones is `sequential` should be ignored, too. I always end up doing weird stuff when I get confused between non-curried parameters and actual tuples.